src/components/TeamMembers/TeamMembers.js

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+
+import TeamMember from '../TeamMember';
+import './TeamMembers.css';
+
+function TeamMembers({ members }) {
+    return(
+        <div className="team-members-wrapper" id="members">
```

Comment:
Jeśli nie używasz do niczego konkretnego tego ID to raczej bym usunął, zbędny kod.

Reply:
To ID do przewijania strony :)